packages/graphql-testing/README.md

```diff
@@ -0,0 +1,49 @@
+# `@shopify/graphql-testing`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Fgraphql-testing.svg)](https://badge.fury.io/js/%40shopify%2Fgraphql-testing.svg)
+
+Utilities to create mock graphql factory.
```

Comment:
This readme doesn't really say what this mock GraphQL object even does, we should definitely add a bit about what it has (`.client` for the Apollo Client, most notably), and what it actually does with GraphQL requests.